### ISSUE TITLE
Specify agent when starting jobs

### ISSUE THREAD

**Issue by mentalblock** [coreos/fleet#406]
I would like to see an enhancement to Fleet that allows me to specify the machine which will run the job without hardcoding it into the unit file. For example, `fleetctl start unit <optional-machine-id>`.

I believe this will solve two problems. This will allow me to override any decision the forthcoming scheduling algorithm would make. In the current situation where we do not have a scheduling algorithm, it will also allow me to manually offset any disproportionate distribution of jobs, as demonstrated in #372, by creating all new jobs on the least loaded machine.

I think that the fleet-specific options in the unit files documented at https://github.com/coreos/fleet/blob/master/Documentation/unit-files.md, namely X-ConditionMachineID, is not sufficient to handle this use case for the loss of the hardcoded machine specified with this option will preclude the job from being executed on other remaining members of the cluster.

In summary, this enhancement would allow the scenario in the cluster be "This unit can picked up and started by any machine, but force start it on machine X this time".
